.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -474,6 +475,8 @@ stages:
 
     - job: native
       timeoutInMinutes: 60 #default value
+      pool:
+        vmImage: windows-2019
```

Comment:
Did you mean to use the scale set for this one too?

```suggestion
        name: azure-windows-scale-set
```

Reply:
no, there's an issue with building the native profiler on the windows scale set currently, as it makes assumptions about the NuGet.exe call. I need to fix the way the profiler does a restore to use the built-in helpers instead, then we can use it!

Reply:
Sounds good, thanks for the explanation!

Reply:
nit -  you could have added a small comment explaining this.